### PR TITLE
Removed unnecessary section in intro to eth

### PR DESCRIPTION
**Pull request by helloitsm3**

### CODE REVIEW

**Comment by AlmostEfficient** [comment]
Curious as to why this is unnecessary 🤔

**Comment by helloitsm3** [comment]
> Curious as to why this is unnecessary 🤔

Sorry my bad. I've added back the section and updated the snippet to include that part.

**Comment by AlmostEfficient** [comment]
I am now even less sure what this PR is doing

**Comment by helloitsm3** [comment]
> I am now even less sure what this PR is doing 

getAllWaves is created but not used anywhere

File: Solidity_And_Smart_Contracts/en/Section_3/Lesson_1_Storing_Messages_From_Users.md [hunk 1]
```diff
@@ -175,77 +175,80 @@ So, here's the new function I added to `App.js`.
...
+  useEffect(async () => {
+    const account = await findMetaMaskAccount()
+
+    // Checks if metamask is connected
+    if (account !== null) {
+      
+    }
+  }, [])
+
+  return (
```
Comment (AlmostEfficient) [inline Solidity_And_Smart_Contracts/en/Section_3/Lesson_1_Storing_Messages_From_Users.md:246]: Why bring this bit back?

Comment (AlmostEfficient) [reply Solidity_And_Smart_Contracts/en/Section_3/Lesson_1_Storing_Messages_From_Users.md:246]: I feel like people will copy paste the whole thing and then realise something is broken but not know what

Comment (helloitsm3) [reply Solidity_And_Smart_Contracts/en/Section_3/Lesson_1_Storing_Messages_From_Users.md:246]: Okay. How about we close this PR for now. Will come back to it if needed.

**Pull request closed without merging**